src/structure_dump.cpp

```diff
@@ -424,7 +424,7 @@ binary_partition::binary_partition(const split_plane &_split_plane,
   if (!left || !right) { meep::abort("Binary partition tree is required to be full"); }
 }
 
-binary_partition::binary_partition(const binary_partition& other) : proc_id{other.proc_id}, plane{other.plane} {
+binary_partition::binary_partition(const binary_partition& other) : proc_id{other.proc_id}, plane(other.plane) {
```

Comment:
Why are you using braces for `proc_id` and parens for `plane`?

Reply:
Interesting discussion here: https://probablydance.com/2013/02/02/the-problems-with-uniform-initialization/ … 

It's a stylistic question, but since we don't use "uniform initialization" elsewhere, and it is apparently exactly equivalent to old-style paren initialization, and it seems to occasionally cause problems with some compiler, maybe it is better to stick with parens

Reply:
```suggestion
binary_partition::binary_partition(const binary_partition& other) : proc_id(other.proc_id), plane(other.plane) {
```

Reply:
Ok